src/app/parcels/ActionBar/Actions.tsx

```diff
@@ -35,7 +36,7 @@ type AvailableActionsType = {
     };
 };
 
-export const availableActions: AvailableActionsType = {
+export const allAvailableActions: AvailableActionsType = {
```

Comment:
I know you didn't set this up :). Why were the actions set to be a key-value pair, in their only use Lines 149 and 173 we do Object.Entries which turns the actions from a key-value pair to an array. Would it make more sense to be an array in the first place, so we don't need the transition? 
e.g. [{actionName: "Download Day Overview", newStatus: "Day Overview Downloaded"}] 
Feel free to ignore this if you don't think it's neater - or there is a good reason for it to be done like this :)

Reply:
Good point 👍 I've changed it into an array